### ISSUE TITLE
picard broken at run-time: Could not find the Qt platform plugin "xcb" in ""

### ISSUE THREAD

**Issue by peti** [NixOS/nixpkgs#65776]
~~~
$ picard 
qt.qpa.plugin: Could not find the Qt platform plugin "xcb" in ""
This application failed to start because no Qt platform plugin could be initialized. Reinstalling the application may fix this problem.
~~~

This is Nixpkgs `master` @ a8d45e7fc3656895fc26ff334055a8a27327f96f. Might be an issue similar to https://github.com/NixOS/nixpkgs/issues/42893. Note, however, that `picard` used to work just fine only a few weeks ago (0ad1b80a3b1536c4b373dd1b5fcf42a3e41bbbd8, for example).

**Comment by worldofpeace** [comment]
needs fixes in #65399

**Comment by worldofpeace** [comment]
Note to fix this you'd need to use `wrapQtApp` manually since `wrapQtAppsHook` ignores non elf files.
(see latest qt docs)

And it can still have issues like in`qutebrowser`( #65290) because `pyqt` is out of date.

###### note the duplicate issues for qutebrowser...

**Comment by FRidh** [comment]
Fixed in https://github.com/NixOS/nixpkgs/pull/65836

**Comment by peti** [comment]
> Fixed in #65836

Unfortunately, it is not. Using revision 0fd978baf77e15c4eb9ba7a54111c340b0730b8f I get:

~~~
$ nixpkgs-build -A picard
/nix/store/vkkfivbr5fwiwwzg7v4gxmbbx0cv66xy-picard-2.1.3
$ /nix/store/vkkfivbr5fwiwwzg7v4gxmbbx0cv66xy-picard-2.1.3/bin/picard
qt.qpa.plugin: Could not find the Qt platform plugin "xcb" in ""
This application failed to start because no Qt platform plugin could be initialized. Reinstalling the application may fix this problem.

zsh: abort      /nix/store/vkkfivbr5fwiwwzg7v4gxmbbx0cv66xy-picard-2.1.3/bin/picard
~~~

**Comment by FRidh** [comment]
That's unfortunate. Wrapping is indeed still needed (https://github.com/NixOS/nixpkgs/issues/65399).
